Sources/SwiftKuery/Utils.swift

```diff
@@ -39,6 +39,9 @@ struct Utils {
         case let val as Parameter:
             return try val.build(queryBuilder: queryBuilder)
         case let value as Date:
+            if let dateFormatter = queryBuilder.dateFormatter {
+                return dateFormatter.string(from: value)
+            }
             return "'\(String(describing: value))'"
```

Comment:
MySQL fails with the `String(describing: value)` representation of Date. Not sure if relying on `String(describing:)` behavior is the right solution for any database.